backend/FwLite/FwLiteProjectSync.Tests/SyncTests.cs

```diff
@@ -98,15 +133,19 @@ public async Task UpdatingAnEntryInEachProjectSyncsAcrossBoth()
 
         var crdtEntries = await crdtApi.GetEntries().ToArrayAsync();
         var fwdataEntries = await fwdataApi.GetEntries().ToArrayAsync();
-        crdtEntries.Should().BeEquivalentTo(fwdataEntries);
+        crdtEntries.Should().BeEquivalentTo(fwdataEntries,
+            options => options
+                .For(e => e.Components).Exclude(c => c.Id)
+                .For(e => e.Components).Exclude(c => c.ComponentHeadword)
```

Comment:
This is due to the fact that we don't yet ever change the headword on components or complex forms in CRDTs?

Reply:
That's right. I'll add a todo though since hopefully we can fix that